source/Octopus.TestPortForwarder/SocketPump.cs

```diff
@@ -19,13 +20,17 @@ public class SocketPump
         readonly TimeSpan sendDelay;
 
         IDataTransferObserver dataTransferObserver;
+        readonly int delaySendingLastNBytes;
```

Comment:
Nit: This variable name threw me a little when I first saw it in the other files. First I thought it was a 'delay' (and should be a TimeSpan), then it sounded like a bool (should I delay sending last n bytes?). Then I saw it was the 'last n bytes' for 'delay sending'.

Naming is hard. Maybe an alternative could be `lastNBytesToDelaySending`?